.github/auto_assign.yml

```diff
@@ -0,0 +1,5 @@
+# Set to true to add reviewers to pull requests
+addReviewers: false
+
+# Set to true to add assignees to pull requests
```

Comment:
The comment suggests that this is a Boolean attribute, but it is set to author. Is the comment wrong?